tests/test_rf_base.py

```diff
@@ -231,6 +231,32 @@ def _forward_step(*, model: _Net, extern_data: TensorDict):
     run_model(extern_data, lambda *, epoch, step: _Net(), _forward_step)
 
 
+def test_linear():
```

Comment:
The name of the test is misleading. This is not about testing `linear`. I guess you want to test weight dropout and/or gradient checkpointing here.